bugshaker/src/main/java/com/github/stkent/bugshaker/flow/email/EmailCapabilitiesProvider.java

```diff
@@ -54,20 +54,6 @@ public EmailCapabilitiesProvider(
         this.logger = logger;
     }
 
-    public boolean canSendEmails() {
-        logger.d("Checking for email apps...");
```

Comment:
We need this method back for devices which do not have email sending abilities or even otherwise.